openfecwebapp/templates/partials/candidates-filter.html

```diff
@@ -19,7 +19,7 @@
       <ul>
         <li>
           <input id="cycle-checkbox-five-thousand" name="five_thousand_flag" type="checkbox" value="true">
```

Comment:
The `name` attribute needs to change to `has_raised_funds` as well, and I would also change the `id` attribute to reflect this (so make it `cycle-checkbox-has-raised-funds`).